projects/json-extract-summery/flow.yaml

```
@@ -0,0 +1,70 @@
+    needs:
+      - triggerNode_1
+  - nodeId: LLMNode_835
+    nodeType: LLMNode
+    nodeName: Generate Text
+    values:
+      tools: []
+      prompts:
+        - id: 187c2f4b-c23d-4545-abef-73dc897d6b7b
+          role: system
+          content: |-
+            You are an AI Assistant  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)